Sources/Afterpay/Resources/ModalLinkStyle.swift

```diff
@@ -34,9 +34,9 @@ public enum ModalLinkStyle {
         attributes: [NSAttributedString.Key.underlineColor: UIColor.clear]
       )
     case .moreInfoText:
-      return ModalLinkConfig(text: Strings.moreInfo)
+      return ModalLinkConfig(text: Afterpay.string.localised.moreInfo)
     case .learnMoreText:
-      return ModalLinkConfig(text: Strings.learnMore)
+      return ModalLinkConfig(text: Afterpay.string.localised.learnMore)
```

Comment:
I wonder if we should use US English as a convention for variable naming. Are these properties public? Have we used British English already elsewhere?

Reply:
Addressed in [8f196d0](https://github.com/afterpay/sdk-ios/pull/207/commits/8f196d03f2504520b355b0714154a32340141aae)